src/main/java/com/github/hcsp/encapsulation/Cat.java

```diff
@@ -18,7 +20,18 @@
      * @param name 名字
      * @return 创建的猫
      */
-    public static Cat newCat(String name, int age) {}
+    public static Cat newCat(String name, int age) {
+        if("".equals(name) || name.equals(null) || age<0){
```

Comment:
```suggestion
        if("".equals(name) || name==null || age<0){
```

`name`如果为`null`的时候，这里会产生空指针。

Reply:
谢了，Ber哥